lib/tasks/trigger_ea_events.rake

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+require 'rake'
+
+namespace :trigger do
+  desc 'Send follow-up emails LDUs two weeks before handover if COM still not assigned'
```

Comment:
This description is wrong (although I can tell which rake task you forked this from 😆)